### ISSUE TITLE
Source Control View

### ISSUE THREAD

**Issue by aarroisi** [zed-industries/community#2242]
### Check for existing issues

- [X] Completed

### Describe the feature

I would like to request a Source Control View, like the one in VSCode where I can see all the changes and staged changes I made in the project. This is very useful for me to review my work before submitting a PR. Maybe you can add a button to open a tab consisting of this information at the bottom left of the screen, right next to the button to open Project Diagnostic. That would be very helpful.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="420" alt="Screenshot 2023-11-08 at 10 12 53" src="https://github.com/zed-industries/community/assets/34854015/57cc85ea-aa84-4f26-aa3e-0a2cdaf4cee9">

**Comment by Otterpocket** [comment]
Hi, I came from Vscode and heavily relied on the above feature. Since using zed I switched to github desktop and honestly, I feel like my workflow/productivity has improved. Sure the inline diff is essential and some kind of inline blame would be nice.
But is version control really the responsibility of the editor? My personal opinion is that I would rather see energy put into things that are clearly the responsibility of the editor.

**Comment by JosephTLyons** [comment]
Yes! It makes sense we would build out a new panel (a git panel) that lives in one of the docks. We definitely want to support git operations! :)
